src/Fixer/LanguageConstruct/FunctionToConstantFixer.php

```diff
@@ -29,12 +29,12 @@
 final class FunctionToConstantFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
     /**
-     * @var string[]
+     * @var array<string, array<int, Token>>
```

Comment:
```diff
-* @var array<string, array<int, Token>>
+* @var array<string, Token[]>
```

to indicate that numeric index is 0,1,2,3... and not 5,8,124,6 ?
same for later